cms-YYYY-validated-runs/code/vali_records.py

```diff
@@ -10,19 +10,46 @@
 
 """
 Create validated data records.
-"""
 
-RECID_START = 14212
-YEAR_RELEASED = 2015
+Input arguments
+- recid (May 2023, the last used recid was 14210 for 2015 pp)
+- year to be released
+- run era (for pp, give one of the released eras, it is needed to get the collision energy, stored per era)
+- type (pp, pPb, pphiref, PbPb)
+
+Inputs:
+- 2013 pPb: 14216 2013 HIRun2013 pPb
+- 2013 ppref: 14218 2013 Run2013A pphiref
+- 2015 ppref: 14212 2015 Run2015E pphiref
+- 2015 pp:  14210 2015 Run2015D pp (not used to build the current record, but tested to produce the same files)
+
+"""
 
-def create_record(recid, year, filename):
+def create_record(recid, year, era, runtype, filename):
     """Create record for the given year."""
 
     rec = {}
 
+    year=str(year)
     year_created = year
-    year = str(year)
```

Comment:
We usually assume having access to `click` library in the other data curation scripts, so if we change the `sys.argv` business to using `click`, then expression such as `year = str(year)` should not be necessary anymore.

This obviously assumes that the script is run in a Python environment that has `click` installed. Do you think we can take this assumption for thsi script too? If yes, I could change the respective parts. Otherwise it's good to go like this too.

Reply:
OK, it would be good to add a `requirements.txt` in that case (maybe there are other packages to install, I'm not sure if all the others are included) and update README. 

In the other PR (#172) `year` is used also in an[ int comparison](https://github.com/cernopendata/data-curation/blob/cms-2013-hi-luminosity-tables/cms-YYYY-luminosity/code/lumi_records.py#L66). Would that work? And are the arguments passed in the same way with `click` e.g. in the []([workflow](https://github.com/cernopendata/data-curation/blob/cms-2013-hi-luminosity-tables/.github/workflows/cms-luminosity-tables.yaml#L89)) (thinking of similar changes in the corresponding code)